lib/lnc.ts

```diff
@@ -78,6 +78,38 @@ export default class LNC {
         );
     }
 
+    get status() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientStatus &&
+            this.wasm.wasmClientStatus()
+        );
+    }
+
+    get expiry() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientGetExpiry &&
+            this.wasm.wasmClientGetExpiry()
+        );
+    }
+
+    get hasPerms(permission: string) {
```

Comment:
This isn't syntactically valid TS. You can't add parameters to a `get` property. It should be a normal function.

Also, there are a few build errors with these new fields. Can you test to make sure they all work? Maybe add more buttons to the kitchen-sink.